js/localgov-publications.js

```diff
@@ -15,3 +15,54 @@
     }
   };
 })(jQuery, Drupal);
+
+/**
+   * Collapse able menu for Publication page
+   *
+   * @param {object} context
+   */
+Drupal.behaviors.publicationMenuToggle = {
+  attach: function(context) {
+    const menuCollapseBreakpoint = 992;
+
+    var headers = [
+      $('.bfc-publication-navigation__content-header', context),
+      $('.bfc-publication-tableofcontent__content-header', context)
+    ];
+    var menus = [
+      $('#block-bfc-publicationnavigation ul.list--no-style', context),
+      $('#block-bfc-publicationstableofcontentsblock .publication-content-block', context)
+    ];
+    function toggleMenuVisibilityAndIcon(header, menu) {
+      header.on('click', function () {
+        menu.toggleClass('is-hidden');
+        header.toggleClass('up-icon down-icon');
+      });
+    }
+
+    let previousWidth = -1;
+
+    function initializeStateForMenus() {
+      const previousCollapseState = previousWidth < menuCollapseBreakpoint;
+      const newCollapseState = window.innerWidth < menuCollapseBreakpoint;
+
+      if (previousCollapseState === newCollapseState && !(previousWidth === -1)) {
+        return;
+      }
+      previousWidth = window.innerWidth;
+
+      headers.forEach(function(header, index) {
+        menus[index].toggleClass('is-hidden', newCollapseState);
+        header.toggleClass('up-icon', !newCollapseState).toggleClass('down-icon', newCollapseState);
+      });
+    }
+    headers.forEach(function(header, index) {
+      if (!header.data('menuToggleAttached')) {
+        toggleMenuVisibilityAndIcon(header, menus[index]);
+        header.data('menuToggleAttached', true);
+      }
+    });
+    initializeStateForMenus();
```

Comment:
We'll need a way to make this feature optional. It can be defaulted to on for new installs, but we can't enable it by default for existing sites.

Reply:
This could be achieved by adding a toggle in the block config for setting enabled/disabled?